libs/sdk-ui-charts/src/highcharts/adapter/HighChartsMeasuredRenderer.tsx

```diff
@@ -0,0 +1,32 @@
+// (C) 2007-2018 GoodData Corporation
+import React from "react";
+import Measure, { MeasuredComponentProps } from "react-measure";
+
+export class HighChartsMeasuredRenderer extends React.PureComponent {
+    private renderChildren(contentRect: any) {
+        return React.Children.map(this.props.children, (child) => {
+            if (React.isValidElement(child)) {
+                return React.cloneElement(child, { contentRect });
```

Comment:
Can we please somehow avoid `cloneElement`? It bit us several times, because it does not type-check the children.

Reply:
If avoiding it is not possible, at least please comment on the prop in HighChartsRenderer so that we know where it comes from.

Reply:
Fixed with 8280e87, @no23reason please check if this solution si better (introduces fn prop)

Reply:
Yes that is much better, thanks :)